pkg/sfu/forwardstats.go

```diff
@@ -0,0 +1,72 @@
+package sfu
+
+import (
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/livekit/livekit-server/pkg/telemetry/prometheus"
+	"github.com/livekit/protocol/utils"
+)
+
+type ForwardStats struct {
+	lock       sync.Mutex
+	lastLeftMs atomic.Int64
+	latency    *utils.LatencyAggregate
+	closeCh    chan struct{}
+}
+
+func NewForwardStats(latencyUpdateInterval, reportInterval, latencyWindowLength time.Duration) *ForwardStats {
+	s := &ForwardStats{
+		latency: utils.NewLatencyAggregate(latencyUpdateInterval, latencyWindowLength),
+		closeCh: make(chan struct{}),
+	}
+
+	go func() {
```

Comment:
just a style comment, can you put this is a function of its own and call it `worker` or something. I think that will improve readability.